### PR TITLE
Enable all snippets and custom commands to work with tsx source files

### PR DESCRIPTION
**Pull request by hoanhtien**
Related to https://github.com/Microsoft/TypeScript-Sublime-Plugin/issues/352.

### CODE REVIEW

File: Default (Linux).sublime-keymap [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "command": "typescript_paste_and_format",
         "context": [
             { "key": "setting.typescript_auto_format", "operator": "equal", "operand": true },
-            { "key": "selector", "operator": "equal", "operand": "source.ts" }
+            { "key": "selector", "operator": "equal", "operand": "source.ts, source.tsx" }
```
Comment (billti) [inline Default (Linux).sublime-keymap]: Interesting. I didn't realize you could just use a comma separated list here. That's handy! :smile:

File: TypeScriptReact.sublime-settings [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "auto_complete_triggers" : [ {"selector": "source.tsx", "characters": "."} ],
```
Comment (billti) [inline TypeScriptReact.sublime-settings]: Is there a reason this isn't just added to the existing sublime-settings file? (Not that it matters much).

**Comment by billti** [comment]
:+1: Thanks!

File: TypeScriptReact.sublime-settings (hunk 2 above)
Comment (hoanhtien) [inline TypeScriptReact.sublime-settings]: It doesn't work. I wasted some hours trying to make it work and eventually found the reason [here](http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/customization/settings.html):

>file type settings need to carry the name of the .tmLanguage syntax definition for the file type. Thus, for the .py file type, whose syntax definition is contained in Python.tmLanguage, the corresponding settings files would be called Python.sublime-settings.

Comment (billti) [inline TypeScriptReact.sublime-settings]: Ah! That would be why I couldn't get it to work :confused: Thanks!

**Pull request merged**